packages/esm-service-queues-app/src/config-schema.ts

```diff
@@ -82,6 +82,19 @@ export const configSchema = {
       _description: 'The UUID of the default status for attending a service in the queues eg In Service.',
       _default: 'ca7494ae-437f-4fd0-8aae-b88b9a2ba47d',
     },
+    defaultUrgentPriorityStatus: {
+      _type: Type.ConceptUuid,
+      _description: 'The UUID of the default priority for a service in the queues eg urgent.',
+      _default: 'ef5718cd-d6d5-4fcf-bb82-cc4afaa90603',
```

Comment:
Please remove the default uuid. This should be blank by default to avoid problems with other implementations

Reply:
I have removed it